manifests/supervisorcluster/1.30/cns-csi.yaml

```diff
@@ -307,7 +307,7 @@ spec:
       priorityClassName: system-node-critical
       containers:
         - name: csi-provisioner
-          image: localhost:5000/vmware/csi-provisioner/csi-provisioner:v5.0.2_vmware.10
+          image: localhost:5000/vmware/csi-provisioner/csi-provisioner:v5.0.2_vmware.12
```

Comment:
why do we bump from csi-provisioner:v5.0.2_vmware.10 to csi-provisioner:v5.0.2_vmware.12?
csi-provisioner:v5.0.2_vmware.11 is skipped?

Reply:
Discussed with @skogta  
csi-provisioner:v5.0.2_vmware.11 was for bumping up CSI provisioner to add annotation for backing disk type and csi-provisioner:v5.0.2_vmware.12 is for revert.